pkg/feeds/getter_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package feeds
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"testing"
+
+	soctesting "github.com/ethersphere/bee/v2/pkg/soc/testing"
+	mockstorer "github.com/ethersphere/bee/v2/pkg/storer/mock"
+)
+
+func TestGetWrappedChunk(t *testing.T) {
+	storer := mockstorer.New()
+
+	// new format (wraps chunk)
+	ch := soctesting.GenerateMockSOC(t, []byte("data")).Chunk()
+	wch, err := GetWrappedChunk(context.Background(), storer.ChunkStore(), ch)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !bytes.Equal(wch.Data()[8:], []byte("data")) {
+		t.Fatal("data mismatch")
+	}
+
+	// old format (ts + ref)
```

Comment:
does the format change for longer, encrypted references? maybe we can add a test case for that?